docs/Makefile

```diff
@@ -13,11 +13,12 @@ help:
 
 .PHONY: help Makefile
 
-tools/wptserve:
+tools/%:
 	mkdir -p $(shell dirname $@)
-	ln -s ../../tools/wptserve $@
+	test -d ../$@
```

Comment:
Did you mean `.././$@`?

Reply:
The value `../$@` is intentional because the `test` command is evaluated in the `docs/` directory. The invocation of `ln` uses an additional "parent" reference (`../../$@`) because it is creating a link in `docs/tools/`.